<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>740</width>
      <height>270</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab1">
     <attribute name="title">
      <string>떡볶이</string>
     </attribute>
     <widget class="QPushButton" name="btn_milddeok">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>40</y>
        <width>230</width>
        <height>160</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/food/밀떡-1.jpg);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_ssalddeok">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>40</y>
        <width>230</width>
        <height>160</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/food/쌀떡-1.jpg);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab2">
     <attribute name="title">
      <string>순대</string>
     </attribute>
     <widget class="QPushButton" name="btn_squidsundae">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>40</y>
        <width>230</width>
        <height>160</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/food/오징어 순대-1.jfif);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_chapsundae">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>40</y>
        <width>230</width>
        <height>160</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/food/찹쌀순대 -1.jpg);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab3">
     <attribute name="title">
      <string>튀김</string>
     </attribute>
     <widget class="QPushButton" name="btn_shrimptuigim">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>40</y>
        <width>230</width>
        <height>160</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/food/새우 튀김-1.jfif);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_gimmari">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>230</width>
        <height>160</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/food/김말이 튀김-1.jfif);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_squidtuigim">
      <property name="geometry">
       <rect>
        <x>251</x>
        <y>40</y>
        <width>230</width>
        <height>160</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/food/오징어 튀김-1.jfif);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QListView" name="foodList">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>340</y>
      <width>371</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>340</y>
      <width>291</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>D2Coding</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>총 가격</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="totalPrice">
       <property name="font">
        <font>
         <family>D2Coding</family>
        </font>
       </property>
       <property name="text">
        <string>0원</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_pay">
       <property name="font">
        <font>
         <family>D2Coding</family>
         <weight>75</weight>
         <bold>true</bold>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 170, 127);</string>
       </property>
       <property name="text">
        <string>PAY</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_reset">
       <property name="font">
        <font>
         <family>D2Coding</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 85, 0);</string>
       </property>
       <property name="text">
        <string>RESET</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resource_1.qrc"/>
 </resources>
 <connections/>
</ui>
